README.md

```diff
@@ -17,6 +17,14 @@ We recommend using the [latest release](https://github.com/EBIvariation/vcf-vali
 
 ## Run
 
+#### Note: Core dumped in Ubuntu 18
+We recently noticed that the current version 0.9.1 fails to execute in Ubuntu 18 with the next message:
```

Comment:
To avoid having to update it until we fix it:

```suggestion
We recently noticed that the current version fails to execute in Ubuntu 18 with the next message:
```